spectests/mainnet_test.go

```diff
@@ -0,0 +1,283 @@
+package spectests
+
+import (
+	"bytes"
+	"io/ioutil"
+	"path"
+	"testing"
+
+	"github.com/bazelbuild/rules_go/go/tools/bazel"
+	"github.com/prysmaticlabs/go-ssz"
+)
+
+func TestSSZStatic_Mainnet(t *testing.T) {
+	testFolders, testsFolderPath := testFolders(t, "mainnet", "ssz_static")
+	for _, folder := range testFolders {
+		eth2TypeName := folder.Name()
+		t.Run(eth2TypeName, func(t *testing.T) {
+			sszCases, err := bazel.Runfile(path.Join(testsFolderPath, eth2TypeName))
+			if err != nil {
+				t.Fatalf("Failed to read file: %v", err)
+			}
+			innerSSZFolders, err := ioutil.ReadDir(sszCases)
+			if err != nil {
+				t.Fatalf("Failed to read file: %v", err)
+			}
+			for _, innerFolder := range innerSSZFolders {
+				innerPath := path.Join(sszCases, innerFolder.Name())
+				testCases, err := ioutil.ReadDir(innerPath)
+				if err != nil {
+					t.Fatalf("Failed to read file: %v", err)
+				}
+				for _, cs := range testCases {
+					serialized, err := bazelFileBytes(path.Join(innerPath, cs.Name(), "serialized.ssz"))
+					if err != nil {
+						t.Fatal(err)
+					}
+					switch folder.Name() {
+					case "AggregateAndProof":
```

Comment:
Can you add a default case that fails? This way we know all cases / folders were covered.